libcloud/compute/drivers/dimensiondata.py

```diff
@@ -2786,6 +2786,10 @@ def _to_node(self, element):
                  state=node_state,
                  public_ips=[public_ip] if public_ip is not None else [],
                  private_ips=[private_ip] if private_ip is not None else [],
+                 size=self.list_sizes()[0],
+                 image=NodeImage(extra['sourceImageId'],
```

Comment:
It would also make it a bit easier to read / fail safe if we use keyword arguments when calling a constructor, but that's not a blocker for me.